### PR TITLE
Minor HTTP20Connection changes

### PR DESCRIPTION
**Pull request by irvind**
I don't know if these tiny changes are appropriate, but here they are.

### CODE REVIEW

File: hyper/http20/connection.py [hunk 1]
```diff
@@ -507,7 +507,6 @@ def _new_stream(self, stream_id=None, local_closed=False):
         """
         Returns a new stream object for this connection.
         """
-        window_size = self._settings[SettingsFrame.INITIAL_WINDOW_SIZE]
```
Comment (Lukasa) [inline hyper/http20/connection.py]: I think this being missing is a bug: looks like it should be the argument to `__wm_class()`

File: hyper/http20/connection.py [hunk 2]
```diff
@@ -723,6 +722,8 @@ def _recv_cb(self):
             except ConnectionResetError:
                 break
 
+            count -= 1
+
```
Comment (Lukasa) [inline hyper/http20/connection.py]: This one looks legit though.

**Comment by Lukasa** [comment]
One of these seems like it is a mistake that leads to a bug, and should be fixed properly. The other one is fixing a bug, though it'd be good to have a test for it.

**Comment by irvind** [comment]
Checked.

**Comment by Lukasa** [comment]
\o/ Thanks @irvind!

**Pull request merged**